### PR TITLE
[UXIT-1706] Enhance Clarity and Consistency in ‘The Blockchain’ Section

### PR DESCRIPTION
**Pull request by mirhamasala**
This PR improves readability and consistency in The Blockchain section of the Filecoin documentation, including updates to language flow, terminology alignment, and clearer explanations.

### CODE REVIEW

File: basics/the-blockchain/actors.md [hunk 1]
```diff
@@ -1,133 +1,141 @@
...
-Like the state tree, a Merkle Directed Acyclic Graph (Merkle DAG) is used to map the set of messages for a given block. Nodes in the messages may contain information on:
+Messages are the instructions actors use to interact with each other. These are also represented in a Merkle DAG. Nodes in the messages may contain:
 
-* The actor the message was sent to
-* The actor that sent the message
-* Target method to call on the actor being sent the message
-* A cryptographic signature for verification
-* The amount of FIL transferred between actors
+- The actor the message was sent to.
+- The actor sending the message.
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:49]: The past tense is more accurate as the message is sent before it is accepted by the network.

File: basics/the-blockchain/actors.md [hunk 2]
```diff
@@ -1,133 +1,141 @@
...
+- The method to call on the receiving actor.
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:50]:
```suggestion
- The method to call on the target actor.
```
Receiving suggests that there is a movement of tokens and not just that an actor may perform an action. 

Think of an actor as a serverless function, you target the function of a particular serverless function not have it receive a method call.

File: basics/the-blockchain/actors.md [hunk 3]
```diff
@@ -1,133 +1,141 @@
...
 
 ### Actor code
 
-The code that defines an actor in the Filecoin network is separated into different methods. Messages sent to an actor contain information on which method(s) to call and the input parameters for those methods. Additionally, actor code interacts with a _runtime_ object, which contains information on the general state of the network, such as the current epoch, cryptographic signatures, and proof validations. Like smart contracts in other blockchains, actors must pay a _gas fee_, which is some predetermined amount of FIL to offset the cost (network resources used, etc.) of a transaction. Every actor has a Filecoin balance attributed to it, a state pointer, a code that tells the system what type of actor it is, and a nonce, which tracks the number of messages sent by this actor.
+Actor code defines what an actor does and is separated into methods. Messages specify which methods to call and any input parameters. Actor code interacts with a _runtime_ object, which provides network information like the current epoch or cryptographic validations. Actors must pay a _gas fee_ (in FIL) for executing operations. Each actor has:
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:56]:
```suggestion
Actor code defines what an actor does and is separated into methods. Messages specify which methods of an actor to call and any input parameters. Actor code interacts with a _runtime_ object, which provides network information like the current epoch, cryptographic signatures, and proof validations. Actors must pay a _gas fee_ (in FIL) for executing operations. Each actor has:
```
Clarifying whose methods and cryptographic validations seemed to be a typo.

File: basics/the-blockchain/actors.md [hunk 4]
```diff
@@ -1,133 +1,141 @@
...
 
 #### InitActor
 
-The `InitActor` can initialize new actors on the Filecoin network. This system actor is instantiated in the genesis block and maintains a table resolving a public key and temporary actor addresses to their canonical ID addresses. The `InitActor` interacts directly with the FVM.
+The `InitActor` initializes new actors on the Filecoin network. It maps public keys and temporary actor addresses to canonical ID addresses. This actor is instantiated in the genesis block and interacts directly with the FVM.
 
 #### AccountActor
 
-The `AccountActor` is responsible for user accounts. Account actors are not created by the `InitActor` but by sending a message to a public-key style address. The account actor updates the state tree with a new actor address and interacts directly with the FVM.
+The `AccountActor` manages user accounts by updating the state tree with new actor addresses. It interacts directly with the FVM.
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:89]: It doesn't `manage` user accounts, it only creates them.

File: basics/the-blockchain/actors.md [hunk 5]
```diff
@@ -1,133 +1,141 @@
...
 
 #### StorageMarketActor
 
-The `StorageMarketActor` is responsible for processing and managing on-chain deals. This is also the entry point of all storage deals and data into the system. This actor keeps track of storage deals and the locked balances of both the client storing data and the storage provider. When a deal is posted on-chain through the `StorageMarketActor`, the actor will first check if both transacting parties have sufficient balances locked up and include the deal on-chain. Additionally, the `StorageMarketActor` holds _Storage Deal Collateral_ provided by the storage provider to collateralize deals. This collateral is returned to the storage provider when all deals in the sector successfully conclude. This actor does not interact directly with the FVM.
+The `StorageMarketActor` processes and manages on-chain deals. It tracks storage deals and the locked balances of both clients and storage providers. When a deal is posted, it ensures both parties have sufficient collateral before including the deal on-chain. Collateral is returned when deals are successfully completed. This actor does not interact directly with the FVM.
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:97]: As far as I am aware the builtin StorageMarketActor is not used for any of the storage deals that occur on the network currently. We should figure out if it should be marked as deprecated or removed. It certainly still exists in the code base but it is not being used as described here and won't be again from my understanding.

File: basics/the-blockchain/actors.md [hunk 6]
```diff
@@ -1,133 +1,141 @@
...
 
 #### StorageMinerActor
 
-The `StorageMinerActor` is created by the `StoragePowerActor` and is responsible for storage mining operations and the collection of mining proofs. This actor is a key part of the Filecoin storage mining subsystem, which ensures a storage miner can effectively commit storage to Filecoin and handles the following:
+The `StorageMinerActor` manages storage mining operations and proof collection. It ensures miners can effectively commit storage and handles:
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:101]:
```suggestion
The `StorageMinerActor` manages storage provider operations and proof collection. It ensures storage providers can effectively commit storage and handles:
```
The actor code still contains the term Miner but it should be changed to StorageProviderActor at some point in line with [FIP0018](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0018.md).

File: basics/the-blockchain/actors.md [hunk 7]
```diff
@@ -1,133 +1,141 @@
...
 
 #### AccountActor
 
-The `AccountActor` is responsible for user accounts. Account actors are not created by the `InitActor` but by sending a message to a public-key style address. The account actor updates the state tree with a new actor address and interacts directly with the FVM.
+The `AccountActor` manages user accounts by updating the state tree with new actor addresses. It interacts directly with the FVM.
 
 #### RewardActor
 
-The `RewardActor` manages unminted Filecoin tokens and distributes rewards directly to miner actors, where they are locked for vesting. The reward value used for the current epoch is updated at the end of an epoch. The `RewardActor` interacts directly with the FVM.
+The `RewardActor` distributes rewards to miner actors based on their contributions. Rewards are locked for vesting, and values are updated at the end of each epoch. This actor interacts directly with the FVM.
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:93]:
```suggestion
The `RewardActor` distributes rewards to storage provider actors based on their contributions. Rewards are locked for vesting, and values are updated at the end of each epoch. This actor interacts directly with the FVM.
```

File: basics/the-blockchain/actors.md [hunk 8]
```diff
@@ -1,133 +1,141 @@
...
 #### PaymentChannelActor
 
-The `PaymentChannelActor` creates and manages _payment channels_, a mechanism for off-chain microtransactions for Filecoin dApps to be reconciled on-chain at a later time with less overhead than a standard on-chain transaction and no gas costs. Payment channels are uni-directional and can be funded by adding to their balance. To create a payment channel and deposit fund, a user calls the `PaymentChannelActor`. This actor does not interact directly with the FVM.
+The `PaymentChannelActor` enables off-chain microtransactions for Filecoin dApps. It manages uni-directional payment channels, allowing transactions to be reconciled on-chain with minimal overhead. This actor does not directly interact with the FVM.
 
 #### StoragePowerActor
 
-The `StoragePowerActor` is responsible for keeping track of the storage power allocated to each storage miner and has the ability to create a `StorageMinerActor`. This actor does not interact directly with the FVM.
+The `StoragePowerActor` tracks the storage power of miners and determines block rewards based on proportional allocations. It creates `StorageMinerActor` instances and does not directly interact with the FVM.
```
Comment (lanzafame) [inline basics/the-blockchain/actors.md:119]:
```suggestion
The `StoragePowerActor` tracks the storage power of storage providers and determines block rewards based on proportional allocations. It creates `StorageMinerActor` instances and does not directly interact with the FVM.
```

**Review by lanzafame — COMMENTED** [review]
@mirhamasala Not sure if you were ready for a review considering the PR was still in draft but since it has been sitting here for 3 weeks I would run an eye over it anyway. Is there anything in particular that you are waiting for to move this PR forward?

**Comment by mirhamasala** [comment]
@lanzafame - Thank you for the review. We can close this, though!

**Pull request closed without merging**